src/schema/nmdc.yaml

```diff
@@ -669,90 +666,88 @@ classes:
           - Required if providing water content
         examples:
           - value: J. Nat. Prod. Plant Resour., 2012, 2 (4):500-503
-          - values: https://dec.alaska.gov/applications/spar/webcalc/definitions.htm
+          - value: https://dec.alaska.gov/applications/spar/webcalc/definitions.htm
         todos:
           - Why is it soil water content method in the name but not the title? Is this slot used in other samples?
           - Soil water content can be measure MANY ways and often, multiple ways are used in one experiment (gravimetric water content and water holding capacity and water filled pore space, to name a few).
           - Should this be multi valued? How to we manage and validate this?
       water_content:
-        annotations:
-          expected_value:
-            tag: expected_value
-            value: string
-          preferred_unit:
-            tag: preferred_unit
-            #value: gram per gram or cubic centimeter per cubic centimeter
-        #multivalued: true
+#        annotations:
+#          expected_value:
+#            tag: expected_value
+#            value: string
+#          preferred_unit:
+#            tag: preferred_unit
+#            #value: gram per gram or cubic centimeter per cubic centimeter
+#        #multivalued: true
         range: string
-        examples:
-          - value: 75%
-          - value: 75 %
-          - value: 0.75 g water/g dry soil
-          - value: 75% water holding capacity
-          - value: 1.1 g fresh weight/ dry weight
-          - value: 10% water filled pore space
+#        examples:
```

Comment:
@turbomam  need some help here. This failed upon 'make all'. What do I need to do here to fix this?

Reply:
works for me. maybe there was something wrong with the indentation. I can show you how to automatically fix some formatting errors like that.